assets/model_monitoring/components/src/model_monitor_compute_histogram_buckets/histogram_buckets.py

```diff
@@ -56,11 +56,12 @@ def compute_histogram_buckets(
     for feature in bin_edges:
         print(bin_edges[feature])
         for i in range(len(bin_edges[feature])):
+            print(type(bin_edges[feature][i]))
             data.append(
                 {
                     "feature_name": feature,
                     "data_type": "Numerical",
-                    "bucket": bin_edges[feature][i],
+                    "bucket": float(bin_edges[feature][i]),
```

Comment:
the type here can only be float?

Reply:
yes above there is a struct which takes only double type as input, in my test case when the value was int it was failing so added explicit cast here